src/extension.ts

```diff
@@ -71,8 +71,8 @@ export function activate(context: vscode.ExtensionContext) {
 
   context.subscriptions.push(MondrianEditorProvider.register(context));
 
-  context.subscriptions.push(PartEditorProvider.register(context));
-  context.subscriptions.push(PartGraphSelPanel.register(context));
+  PartEditorProvider.register(context);
```

Comment:
From clean code's view, A little bit of improvement is that it does registration at one place. (not passing the provider disposable to the caller)